### ISSUE TITLE
Add more semantics to guitar notation

### ISSUE THREAD

**Issue by mdgood** [w3c/musicxml#103]
The combination of MusicXML 3.0's words and bracket elements within a direction element can capture the visual appearance of a wide range of guitar notation. However, as with some of the piano pedal indications described in issue #102, the semantics are lost when using such a general-purpose element.

MusicXML could better capture the semantics behind guitar notation. Possible approaches would be creating new elements at the visual level, creating new elements for playback that could be associated with the direction, or some combination of the two. Some of the specific techniques requested are palm mute, let ring, and vibrato. 

See the MusicXML forum discussion started by Snake and Mogens Lundholm at http://forums.makemusic.com/viewtopic.php?f=12&t=2504, and the earlier MusicXML forum discussion started by Didier Guillion at http://forums.makemusic.com/viewtopic.php?f=12&t=1464.

**Comment by snakebyte69** [comment]
Will these issue be fixed in MusicXML 4.0?

**Comment by dspreadbury** [comment]
No, they are not in the [MusicXML 4.0 milestone](https://github.com/w3c/musicxml/milestone/2).

**Comment by GeorgeTailor** [comment]
How about 4.1? Seems like those features are requested for at least 7 years, would be good to finally standardize them.

**Comment by snakebyte69** [comment]
Will these going to make it into MusicXML 4.1?
